src/pandroid/app/src/main/java/com/panda3ds/pandroid/view/controller/nodes/Joystick.java

```diff
@@ -0,0 +1,115 @@
+package com.panda3ds.pandroid.view.controller.nodes;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import androidx.annotation.NonNull;
+import androidx.appcompat.widget.AppCompatTextView;
+import com.panda3ds.pandroid.math.Vector2;
+import com.panda3ds.pandroid.view.controller.ControllerNode;
+import com.panda3ds.pandroid.view.controller.TouchEvent;
+import com.panda3ds.pandroid.view.controller.listeners.JoystickListener;
+
+public class Joystick extends BasicControllerNode implements ControllerNode {
+	private float stick_x = 0;
```

Comment:
Same note about variable naming, should be stickX/stickY

Reply:
```suggestion
	private float stickX = 0;
```